app/models/application_record.rb

```diff
@@ -30,6 +30,8 @@ def append_url_options(path, options = {})
     path
   end
 
+  def destroy_associated(locale); end
```

Comment:
Can we move this to `TranslatableModel`, I think it makes more sense there.

Reply:
yeah, i did originally, but it causes a tonne of errors because not everything that includes the translation concern uses that model. Have pushed up a quick wip commit but here's the run https://github.com/alphagov/whitehall/actions/runs/8392753788/job/22986130575

Reply:
Try `lib/translatable_model.rb`, that's where `remove_translations_for` lives

Reply:
you're a gentleman and a scholar! Done